lib/std/zig/parser_test.zig

```diff
@@ -7,6 +7,16 @@ test "zig fmt: trailing comma in container declaration" {
         \\const X = struct { foo: i32 align(4) = 1, bar: i32 align(4) = 2 };
         \\
     );
+    try testCanonical(
+        \\test "" {
+        \\    comptime {
+        \\        const X = struct {
+        \\            x: i32
```

Comment:
there's an existing precedent that lack of a trailing comma will cause the struct to be rendered all on one line, and presence of a trailing comma will cause it to be rendered each field on its own line. Function parameters for example.

Reply:
#4352 introduced one more rule, if the opening brace and the first initializer are not on the same line *and* there's no trailing comma all the elements are printed inline but on a separate line.

Reply:
Ah. I actually didn't notice that. I'm a bit resistant to adding even more ways to format things. The way I see it, the limited options is the main feature of zig fmt. I'd like to make a motion to reverse this precedent.